src/charts/donut/donut-pct-chart.html

```diff
@@ -1,3 +1,4 @@
 <span>
-  <div pf-c3-chart id={{donutChartId}} config="config"></div>
+  <div pf-c3-chart ng-if="data.dataAvailable !== false" id={{donutChartId}} config="config"></div>
+  <div pf-empty-chart ng-if="data.dataAvailable === false" chart-height="172"></div>
```

Comment:
@jeff-phillips-18 The Utilization trend chart also has the height hard coded for that parameter because neither of those directives contain a chartHeight field. Should that be added into them?